exercises/faq-page/README.md

```diff
@@ -0,0 +1,112 @@
+# Frequently Asked Questions and Cheat Sheet
+
+> This section contains common information needed when working through the exercises and a command line cheat sheet to help with completing exercises.
+
+---
+
+## Frequently Asked Questions
+
+- How can I get into my CodeReady Workspace (assumes you have gone through Exercise 1, Part 1 where we created the cloud IDE)?
+
+  1. Navigate to the CodeReady workspaces link provided by your facilitators. Take note that this is not the exact same link as in Exercise 1, Part 1 which was used to deploy the cloud IDE. After navigating to the link, you should see a page similar to the picture below.
+
+  ![code-ready-workspaces](../images/faq/codeready-workspace-entrypoint.png)
+
+  2. On the left nav bar under "Recent Workspaces", select the workspace that has `do500-id` (in the picture above, that's `do500-p4ez1`).
+
+  3. The workspace should then start up and when complete you should be back in the cloud IDE.
+
+- How do I get the OpenShift login token?
+
+  1. Login to the cluster at the cluster URL provided.
```

Comment:
Maybe clarify this in case they are reading out of context - cluster URL provided by who?  Maybe a hint to what it would look like?  Who to ask if they are unclear...